.idea/vcs.xml

```diff
@@ -2,10 +2,15 @@
 <project version="4">
   <component name="CommitMessageInspectionProfile">
     <profile version="1.0">
+      <inspection_tool class="BodyLimit" enabled="true" level="WARNING" enabled_by_default="true" />
       <inspection_tool class="GrazieCommit" enabled="true" level="TYPO" enabled_by_default="true" />
+      <inspection_tool class="SubjectBodySeparation" enabled="true" level="WARNING" enabled_by_default="true" />
+      <inspection_tool class="SubjectLimit" enabled="true" level="WARNING" enabled_by_default="true">
+        <option name="RIGHT_MARGIN" value="100" />
+      </inspection_tool>
     </profile>
   </component>
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
   </component>
-</project>
+</project>
```

Comment:
bisogna configurare intellij in modo che aggiunga la newline finale. hai installato il plugin editorconfig?